src/pages/Dashboard/components/DashboardHeader/MobileUserHeader.tsx

```diff
@@ -42,6 +42,24 @@ export const MobileUserHeader = ({
     [mipdProviders, maybeRdns],
   )
 
+  useLayoutEffect(() => {
+    const body = document.body
+    const header = document.querySelector('.mobile-user-header')
+    if (window.visualViewport) {
+      const vv = window.visualViewport
+      const fixPosition = () => {
+        if (body && header) {
```

Comment:
preferably-blocking: declare body and header here to avoid closure issues with this event listener and always have latest